apps/playground/src/app/modules/components/badge-module/badge-module.component.html

```diff
@@ -1,82 +1,117 @@
-<section class="module-description">
-  <h2>Description</h2>
-  <p>Some description</p>
+<section class='module-description'>
+  <h2>DsfrBadgeComponent</h2>
+  <p>
+    The <code>DsfrBadgeComponent</code> purpose is to represent the badge
+  </p>
+  <p>
+    It is based on the
+    <a
+      target='_blank'
+      hreflang='fr'
+      title='The documentation page for the Badge Component in the DSFR'
+      href='https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/badge/'>
+      Badge
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    component in the
+    <a
+      target='_blank'
+      hreflang='fr'
+      title='The DSFR documentation'
+      href='https://www.systeme-de-design.gouv.fr/'>
+      DSFR
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    .
+  </p>
+  <p>It has several <code>@Input()</code> properties:</p>
+  <dl class='input-properties'>
+    <play-input-property-definition
+      name='label'
+      type='It takes a string'
+      [required]='true'>
+    </play-input-property-definition>
+    <play-input-property-definition
+      name='theme'
+      type='It takes a string with many possible values : <code>info</code>, <code>new</code>, <code>warning</code>, <code>success</code>, <code>error</code>
+      , ...'
+      [required]='true'>
+    </play-input-property-definition>
+    <play-input-property-definition
+      name='size'
+      type='It takes a string with 2 possible values : <code>sm</code> or <code>md</code>'
+      default='md'>
+    </play-input-property-definition>
+    <play-input-property-definition
+      name='noIcon'
+      type='boolean'
+      default='false'>
+    </play-input-property-definition>
```

Comment:
`Even though, per DSFR policy, there's no possibility to provide a custom icon, "system" badges will display one by default, as shown in the documentation. Via this property, you can deactivate the display of these system icons.`